Orange/widgets/data/tests/test_owpreprocess.py

```diff
@@ -44,6 +44,30 @@ def test_normalize(self):
         np.testing.assert_allclose(output.X.mean(0), 0, atol=1e-7)
         np.testing.assert_allclose(output.X.std(0), 1, atol=1e-7)
 
+    def test_data_column_nans(self):
+        """
+        ZeroDivisonError - Weights sum to zero, can't be normalized
+        In case when all rows in a column are NaN then it throws that error.
+        GH-2064
+        """
+        table = Table(
```

Comment:
If the same dataset is used in multiple tests, would you consider moving it to [datasets](https://github.com/biolab/orange3/blob/master/Orange/widgets/tests/base.py#L662)?